### PR TITLE
Awesome books

### PR DESCRIPTION
**Pull request by MariamOsman11**
**The setup:**
- Create a new directory and initialize it with git.
- Use npm init -y command to create package.json file.
- Create the entry point for your JavaScript code called index.js
- Create index.css file for your styles.
- Create the main index.html file. Link your index.js and index.css files there.
- Create a directory called modules.

**Re-create your Awesome books app following the rules:**
Use ES6 syntax.
- Divide your code into modules.
- Create a separate file for each module in the modules directory.
- Import modules into the index.js file. 
- Keep all funcionalities of the app without errors.
- Refactor your functions to arrow functions.
- Use let and const in the correct way.
- Display the current date below the navigation bar.

**For this requirement you should use a library:**

- Download the library into your project using npm.
- Import it as a module.

### CODE REVIEW

File: modules/SPA.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  formInfo.classList.add('active');
+  BooksContainer.classList.remove('active');
+  contact.classList.remove('active');
+}
+export function displayContactSection() {
+  contact.classList.add('active');
+  bookslisth2.classList.remove('active');
+  BooksContainer.classList.remove('active');
+  formInfo.classList.remove('active');
+}
```
Comment (julie-ify) [inline modules/SPA.js:23]: - [ ] Kindly change these methods to arrow function per the guidelines for es6 syntax

File: modules/addRemoveBook.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+          <button class="remove-book">Remove</button>
+      `;
+
+  BooksContainer.appendChild(booksInfo);
+};
+
+export function removeBookFromPage(target) {
+  if (target.classList.contains('remove-book')) {
+    target.parentElement.remove();
+  }
```
Comment (julie-ify) [inline modules/addRemoveBook.js:22]: - [ ] Kindly change to arrow function.

File: modules/localStorage.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+export function loadFromLocalStorage() {
+  let Books;
+
+  if (localStorage.getItem('BooksInfo')) {
+    Books = JSON.parse(localStorage.getItem('BooksInfo'));
+  } else {
+    Books = [];
+  }
+  return Books;
+}
```
Comment (julie-ify) [inline modules/localStorage.js:12]: - [ ] Kindly change to arrow function.

File: modules/localStorage.js [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+  const newID = Number(idd);
+  for (let i = 0; i < Books.length; i += 1) {
+    if (Books[i].id === newID) {
+      k = i;
+      break;
+    }
+  }
+  Books.splice(k, 1);
+  localStorage.setItem('BooksInfo', JSON.stringify(Books));
+}
```
Comment (julie-ify) [inline modules/localStorage.js:35]: - [ ] Kindly change to arrow function.

**Review by julie-ify — COMMENTED** [review]
Hi @MariamOsman11,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

* You have submitted a project that does not a descriptive readme file.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 5]
```diff
@@ -1 +1,177 @@
...
+
+Microverse materials
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./mit.md) licensed.
```
Comment (AbelG101) [inline README.md:175]: - [ ] Ouch! this is close but you seem to be missing a license file in your project causing the MIT link in your readme to be broken, so please make sure to resolve this ✔

**Review by AbelG101 — CHANGES_REQUESTED** [review]
# Hi @MariamOsman11 👋, 

You're doing a good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Add and remove functionality works as expected ✔
- Functionalities are divided into modules ✔
- Well-structured files ✔
- All linter checks are passing ✔

## Required Changes ♻️
- _Check the comments under the review._
- you just have to make a few changes and you're on your way 🚀.


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by binhussen — APPROVED** [review]
Hi @MariamOsman11 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

## Highlights
- Add and remove functionality works as expected ✔
- Functionalities are divided into modules ✔
- Well-structured files ✔
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**